pkg/utils/utils.go

```diff
@@ -402,3 +403,14 @@ func ExtractAnnotaions(obj *coreV1.Event) map[string]string {
 
 	return map[string]string{}
 }
+
+//GetClusterNameFromKubectlCmd this will return cluster name from kubectl command
+func GetClusterNameFromKubectlCmd(cmd string) string {
+	i := strings.Index(cmd, "--cluster-name") + 15 //15 indicates 15 letters in --cluster-name string with space or equal
```

Comment:
Please see if we could use valid regex instead.

Reply:
Not as a part of this PR but we should use `cli` parser (like github.com/urfave/cli) to parse BotKube commands.

Reply:
added regex